01 - Community/Obsidian Roundup/2022-01-15  Dataview Optimizations, Live Preview for Mobile, and Community Talks.md

```diff
@@ -0,0 +1,145 @@
+author: Eleanor Konik
+published: 2022-01-15T13:30:00
+publish: true
+---
+
+# 2022-01-15: Dataview Optimizations, Live Preview for Mobile, & Community Talks
+
+## In The Community
+
+-   Obsidian is up for the [Golden Kitty Award](https://www.producthunt.com/golden-kitty-awards-2021/mobile-app) at ProductHunt for Obsidian Mobile. Go vote — and check out the other top contenders, because there's some interesting software being developed and shared over there.
+-   Alexander Rink has been doing performance benchmarks of different "tools for thought." The [results for Obsidian](https://www.goedel.io/p/tft-performance-obsidian) "blew away the competition up to a factor of more than 100." Make sure to read the comments! There's an _excellent_ discussion (featuring plugin dev extraordinaire `@pjeby`) about how to get Obsidian to be able to do all the "outliner" and "block" functionality. Pjeby is very knowledgeable about stuff that not everyone realizes is possible in Obsidian. This comment thread also gives a neat look "under the hood" of how hard Licat has worked to make Obsidian so performant.  Some of the key points have been [migrated to the Obsidian forum](https://forum.obsidian.md/t/understanding-obsidian-and-how-it-works/30603) for posterity.
+-   The AppStories podcast has been doing an in-depth series on Obsidian, and I particularly enjoyed the discussion in [Part 3](https://overcast.fm/+I5ClvB3Dk) about cleaning up no-longer-used plugins and taking a moment to think about how you're using your tools (and what potential conflicts might have snuck into your workflow). Incidentally, here's a [discussion about which Tools for Thought & productivity podcasts are worth a listen](https://twitter.com/thoresson/status/1479836948768477190).
+-   The [plugins by category](https://publish.obsidian.md/hub/02+-+Community+Expansions/02.01+Plugins+by+Category/%F0%9F%97%82%EF%B8%8F+02.01+Plugins+by+Category) list in the Obsidian Hub has far surpassed what I can personally maintain for the Roundup's plugins page, so I took that down. I _highly_ recommend perusing the Hub's plugin list.
+-   The Obsidian Community talks are leveraging the new Discord events features. You can find out more about the upcoming talks about [plugins that won Obsidian October](https://discord.com/events/686053708261228577/928414264950128660) (today) and [themes that won Obsidian October](https://discord.com/events/686053708261228577/928415936963289148) (tomorrow) on Discord.
+-   There's a new [Discord thread](https://discord.com/channels/686053708261228577/700466324840775831/931635167997595748) for sharing ideas of what Obsidian merchandise you'd consider buying. Leah made some beautiful T-shirt/mug mockups if you'd like to get a sneak peek.
+
+## Obsidian Updates
+
+-   Live Preview has come to the Insider build for Mobile. The only [announcement](https://discord.com/channels/686053708261228577/817515900349448202/930508939987124224) I was able to find was in  Discord.
```

Comment:
```suggestion
-   Live Preview has come to the Insider build for Mobile. The only [announcement](https://discord.com/channels/686053708261228577/817515900349448202/930508939987124224) I was able to find was in Discord.
```

Reply:
More trivia: that suggested edit is deleting another double-space making word-wrapping wonky